earth_enterprise/rpms/build.gradle

```diff
@@ -233,7 +233,6 @@ task openGeePostGisRpm(type: GeeRpm) {
     type = BINARY
     autoFindProvides = true
     autoFindRequires = true
-    requires('opengee-common', openGeeVersion, GREATER | EQUAL)
 
```

Comment:
Moved from config phase to early action in task because during the config phase open GEE version isn't known.  It isn't known until after the `setGeeVersion` task is run during the execution phase.